lib/build/helpers/BuildContext.js

```diff
@@ -80,9 +80,9 @@ class BuildContext {
 		return projectBuildContext;
 	}
 
-	async executeCleanupTasks() {
+	async executeCleanupTasks(force) {
 		await Promise.all(this._projectBuildContexts.map((ctx) => {
-			return ctx.executeCleanupTasks();
+			return ctx.executeCleanupTasks(force);
 		}));
```

Comment:
I would suggest to default to `false` in here, so that the callbacks are always called with a boolean.

Reply:
Done